packages/js/data/src/onboarding/types.ts

```diff
@@ -210,6 +210,7 @@ export type Extension = {
 	is_activated?: boolean;
 	learn_more_link?: string;
 	install_priority?: number;
+	requires_jpc?: boolean;
```

Comment:
same with the other PR, could we add a comment here to indicate what JPC means? It will help someone in the in the future to understand what's going on since it's an acronym - I'd prefer the query param be requires_jetpack_connection but that's quite long too

Reply:
Sounds good! Will do!

Reply:
Updated it in [Add description for requires_jpc](https://github.com/woocommerce/woocommerce/pull/52887/commits/483b6df1275a43eba3659b8bc510d5bca9997ac0)